test/Interop/Cxx/foreign-reference/extensions.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -Xfrontend -disable-availability-checking -Onone)
```

Comment:
Since this test doesn't have any executable code, is it effectively a typechecking test? Do you think it would be worth trying to verify, for instance, that a value can be returned from a static function via this mechanism?

Reply:
Ah my bad! I wanted to call some code to make sure the binary does not actually crash at runtime. Will update the test.

Reply:
Test updated.